### PR TITLE
Coding version1

### PR DESCRIPTION
**Pull request by ToscaBeijaert**
This is the first version of the game. The map is loaded, the player can run around, enemies are spawned and do damage. Dying is not possible yet, neither is doing damage.

### CODE REVIEW

File: code/fighter.py [hunk 1]
```diff
@@ -138,6 +153,13 @@ def ai(self, obstacle_list):
 
 
     def draw(self, screen, screen_scroll): # last thing you want to happen
+        """ Draws images on the actual screen
```
Comment (MarayahAy) [inline code/fighter.py:228]: I'm not sure what you mean by 'actual' screen here.

**Review by MarayahAy — COMMENTED** [review]
Your documentations explain every step very well! 
My only comment is that they are all structured a little differently, so an improvement would be to have each documentation be written in the same style and structure. Otherwise great job!!

File: code/fighter.py (hunk 1 above)
Comment (ToscaBeijaert) [reply code/fighter.py:228]: I meant just that it will draw images on the screen, I'll take the ''actual'' away

File: code/fighter.py [hunk 2]
```diff
@@ -107,8 +118,12 @@ def update(self): # TODO: ADD UPDATE FUNCTION
        pass
 
     def ai(self, obstacle_list):
-        # if self.alive and player.alive:
+        """Initialises movement for the enemies
 
+         :param obstacle_list: A list with all obstacles
```
Comment (Sebasdevrs) [inline code/fighter.py]: Are obstacles objects in the map that the enemies can collide with?

**Review by Sebasdevrs — COMMENTED** [review]
Amazing work. Super impressed by the thoroughness of your documentation and by the project in general!

**Comment by ToscaBeijaert** [comment]
> Your documentations explain every step very well!
> My only comment is that they are all structured a little differently, so an improvement would be to have each documentation be written in the same style and structure. Otherwise great job!!

What exactly do you mean with "the documentations are all structured a little differently?" Do you have an example :)

File: code/fighter.py (hunk 2 above)
Comment (ToscaBeijaert) [reply code/fighter.py]: Thx for the comment! I indeed described that rather vaguely, my bad: 
:param obstacle_list: A tuple containing a list of all obstacles. Which is tile information for only things that
         are seen as obstacles (e.g., things the player can walk into like a wall, or the ground). This list includes
          an image of the obstacle and the position of the obstacle.

Does this make it more clear?

**Pull request merged**